src/constants.h

```diff
@@ -104,6 +104,11 @@ constexpr const char* APPLE_SUBSCRIPTIONS_URL =
 constexpr const char* GOOGLE_SUBSCRIPTIONS_URL =
     "https://play.google.com/store/account/subscriptions";
 
+// This is the default time a notification will be shown when received.
+constexpr uint32_t DEFAULT_NOTIFICATION_TIMER = 1 * 1000;  // 1s
```

Comment:
Is this been decided by Peter? 1 sec is really a short time. What do you think?

Reply:
Yeap I agree, I just noticed all the other notifications are using 4s so I'll stick to that.